tasks.py

```diff
@@ -1,101 +1,38 @@
 #!/usr/bin/env python3
 
-import json
 import os
-import subprocess
 import sys
-from typing import Any, List
+from typing import List
 
 from deploykit import DeployGroup, DeployHost
 from invoke import task
 
-RSYNC_EXCLUDES = [
-    ".direnv",
-    ".git",
-    ".mypy-cache",
-    ".ruff_cache",
-    ".terraform",
-    "result*",
-]
 
-
-def deploy_nixos(hosts: List[DeployHost]) -> None:
+@task
+def deploy(c, hosts=""):
     """
-    Deploy to all hosts in parallel
+    Deploy to all servers in parallel. Use inv deploy --hosts build01 to deploy to a single server
     """
-    g = DeployGroup(hosts)
+    g = DeployGroup(get_hosts(hosts))
 
     def deploy(h: DeployHost) -> None:
-        target = f"{h.user or 'root'}@{h.host}"
-        h.run_local(
-            f"rsync {' --exclude '.join([''] + RSYNC_EXCLUDES)} -vaF --delete -e ssh . {target}:/etc/nixos"
+        hostname = h.host.replace(".nix-community.org", "")
+        h.run(
+            [
+                "sudo",
+                "nixos-rebuild",
+                "switch",
+                "--option",
+                "accept-flake-config",
+                "true",
+                "--flake",
+                f"github:nix-community/infra.#{hostname}",
+            ]
         )
 
-        h.run("nixos-rebuild switch --option accept-flake-config true")
-
     g.run_function(deploy)
 
 
-def sfdisk_json(host: DeployHost, dev: str) -> List[Any]:
-    out = host.run(f"sfdisk --json {dev}", stdout=subprocess.PIPE)
-    data = json.loads(out.stdout)
-    return data["partitiontable"]["partitions"]
-
-
-def _format_disks(host: DeployHost, devices: List[str]) -> None:
-    assert (
-        len(devices) == 1 or len(devices) == 2
-    ), "we only support single devices or mirror raids at the moment"
-    # format disk with as follow:
-    # - partition 1 will be the boot partition, needed for legacy (BIOS) boot
-    # - partition 2 is for boot partition
-    # - partition 3 takes up the rest of the space and is for the system
```

Comment:
Maybe it would be good to already write some disko configuration since we otherwise don't have anything to document the current disk layout.

Reply:
Or at least copy out the current code into an issue with a TODO to fix it eventually.

Reply:
I'll take a look at writing the disko config.

Reply:
I've kept a copy of the current config in https://github.com/nix-community/infra/pull/485.